src/app/config/projectProperties.json

```diff
@@ -72,7 +72,7 @@
             ]
         },
         "single-sign-on": {
-            "displayname": "Office Add-in Task Pane project supporting single sign-on",
+            "displayname": "Office Add-in Task Pane project with single sign-on (not for production)",
```

Comment:
Maybe we should say "developer template" (or something like) instead of "not for productions" and then explain a little in our helper text output at the end of project generation.

Reply:
"learning template"? Yes I was considering another PR to the template itself to modify the task pane and readme to make it about learning, but to use the official sample for production.

Reply:
Or maybe just "Sample"

Reply:
What does someone do if they want to use this in production? What's different with the official sample?